scripts/apps/authoring/authoring/directives/ItemCarouselDirective.js

```diff
@@ -0,0 +1,55 @@
+import 'owl.carousel';
+
+/**
+ * @ngdoc directive
+ * @module superdesk.apps.authoring
+ * @name sdItemCarousel
+ *
+ * @requires $timeout
+ */
+
+ItemCarouselDirective.$inject = ['$timeout'];
+export function ItemCarouselDirective($timeout) {
+    return {
+        scope: {
+            items: '='
+        },
+        transclude: true,
+        templateUrl: 'scripts/apps/authoring/views/item-carousel.html',
+        link: function(scope, elem) {
+            let carousel;
+
+            $timeout(() => {
```

Comment:
why is the timeout used there?

Reply:
actually not sure if this part is needed given that there is that `scope.$watch('items')` later